src/main/java/com/mcmoddev/mmdbot/modules/logging/misc/EventReactionAdded.java

```diff
@@ -70,15 +72,37 @@ public void onMessageReactionAdd(final MessageReactionAddEvent event) {
             final List<Long> badReactionsList = getConfig().getBadRequestsReactions();
             final List<Long> goodReactionsList = getConfig().getGoodRequestsReactions();
             final List<Long> needsImprovementReactionsList = getConfig().getRequestsNeedsImprovementReactions();
-            final int badReactions = Utils.getNumberOfMatchingReactions(message, badReactionsList::contains);
-            final int goodReactions = Utils.getNumberOfMatchingReactions(message, goodReactionsList::contains);
-            final int needsImprovementReactions = Utils.getNumberOfMatchingReactions(message,
+            final var badReactions = Utils.getMatchingReactions(message, badReactionsList::contains);
+
+            final var users = badReactions.get(0).retrieveUsers();
+            final var hasStaffSignoff = users.stream().anyMatch(user -> guild.getMember(user).hasPermission(Permission.KICK_MEMBERS));
+
+            final int badReactionsCount = badReactions.stream().mapToInt(MessageReaction::getCount).sum();
+            final int goodReactionsCount = Utils.getNumberOfMatchingReactions(message, goodReactionsList::contains);
+            final int needsImprovementReactionsCount = Utils.getNumberOfMatchingReactions(message,
                 needsImprovementReactionsList::contains);
 
-            final double requestScore = (badReactions + needsImprovementReactions * 0.5) - goodReactions;
+            final double requestScore = (badReactionsCount + needsImprovementReactionsCount * 0.5) - goodReactionsCount;
 
             final User messageAuthor = message.getAuthor();
             if (requestScore >= removalThreshold) {
+                if (!hasStaffSignoff) {
+                    LOGGER.info(REQUESTS, "Request from {} has a score of {}, reaching removal threshold {}, "
+                            + "awaiting moderation approval.",
+                        messageAuthor, requestScore, removalThreshold);
+
+                    final var logChannel = guild.getTextChannelById(getConfig()
+                        .getChannel("events.requests_deletion"));
+                    if (logChannel != null) {
+                        logChannel.sendMessage(String.format("Request from %s (%s;%s) reached deletion threshold "
+                                + "awaiting moderator approval for deletion", messageAuthor.getAsMention(),
+                            messageAuthor.getAsTag(), messageAuthor.getId(), message.getContentRaw()))
```

Comment:
Two things:

1. Add a `,` after the word `threshold`; and
1. Remove the additional `message.getContentRaw()` argument, as there's no corresponding parameter in the format string, or alternatively add a parameter to the format string such as `%.20s` and pass in `message` directly to include a snippet of the message's contents (as per the [javadocs on Message](https://ci.dv8tion.net/job/JDA/javadoc/net/dv8tion/jda/api/entities/Message.html)).

Reply:
Oops, copy paste error, good catch!